tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -13,6 +14,10 @@
 import host_tools.network as net_tools  # pylint: disable=import-error
 import host_tools.drive as drive_tools
 
+from integration_tests.functional.test_vsock import _make_blob, _test_host_connections, _test_guest_connections
```

Comment:
We should find a better solution when importing symbols starting with the underscore character `_`. As stated in the [PEP8 guide](https://pep8.org/#descriptive-naming-styles), symbols prefixed with `_` are an indication that they're designed for internal use.
On top of that, importing symbols between tests is not something that we should do, as it introduces inter-test dependency. 

Ideally, if you regard a testing framework as a tree, the tests should be the leafs that are connected to the underlying branches (in our case framework). If we introduce a dependency between these leafs, when one of them is torn off, the other one needs to be torn off too.

Probably the simplest solution would be to move the common functions in a new file (such as `utils_vsock.py`) so that we reuse code and minimize test length.

What do you think?

Reply:
I like the idea, it makes sense to have a vsock utils file for the shared helper functions.